bwproject.py

```diff
@@ -71,9 +71,11 @@ def _get_auth(self, username, password, token_path, grant_type, client_id):
             self.apiurl + self.oauthpath,
             params={
                 "username": username,
-                "password": password,
                 "grant_type": grant_type,
                 "client_id": client_id
+            },
+            body={
```

Comment:
There is no "body" parameter for requests.post(), should use "data" instead.

Please refer to: [requests/requests/api.py](https://github.com/requests/requests/blob/b5eacad6381a0cd6d7140ece33f76d814fe172e9/requests/api.py#L101)

Reply:
Thank you. Fixed